lib/milc_interface.cpp

```diff
@@ -1557,6 +1557,13 @@ struct mgInputStruct {
 
     /* Setup */
 
+    setup_use_mma[0] = true;
+    setup_use_mma[1] = true;
+    setup_use_mma[2] = true;
+    dslash_use_mma[0] = true;
+    dslash_use_mma[1] = true;
+    dslash_use_mma[2] = true;
+
```

Comment:
@weinbe2 I modified this file according to my understandings but please give the changes to this file a review. :)

Reply:
Looks good to me!